src/cljs/harja/ui/ikonit.cljs

```diff
@@ -718,18 +718,17 @@
 
 (defn ikoni-ja-teksti [ikoni teksti]
   [:span
-   ikoni
-   [:span (str " " teksti)]])
+   [:span.margin-right-8 ikoni]
```

Comment:
Ehkä makuasiakin, mutta mietin pitäskö antaa ns. semanttinen luokka, esim. karttaikoni-ja-teksti, ja less:ssä sitten määrittäis ton margin-right-8:n? Olisko se tyyli helpommin vaihdettavissa jos joskus elää. Ei oo välttämätöntä tätä muuttaa kuitenkaan, tuli vaan mieleen.